### PR TITLE
AV-69603: Added retry for server-check script if cloud-init is not finished

### PR DESCRIPTION
**Pull request by Rishabh-Baranwal**

### CODE REVIEW

File: others/server-check.sh [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
 
-set -e
 set -x
 
+source /root/demo-openrc.sh
+
+sleep_count=0
+while [[ $sleep_count -lt 60 ]];
+do
+    server_cloud_init_status=`openstack console log show server1 | grep Cloud-init | grep 'finished at'`
```
Comment (anantpatil) [inline others/server-check.sh]: I think `grep finished at` at the last record of console log should be enough. Also this will unnecessarily fetch all the logs...
`openstack console log show --lines 10 server1 | grep 'finished at'` 

e.g.
`root@anant2-rocky:~# openstack console log show --lines 10 server1 | grep 'finished at'
Cloud-init v. 0.7.5 finished at Tue, 10 Sep 2019 09:35:21 +0000. Datasource DataSourceConfigDriveNet [net,ver=2][source=/dev/sr0].  Up 275.50 seconds`

File: others/create-nginx-vm.sh [hunk 2]
```diff
@@ -21,6 +21,8 @@ openstack server create --flavor m1.se \
 
 sleep 5
 
+set +e
```
Comment (anantpatil) [inline others/create-nginx-vm.sh]: why is this needed?

Comment (Rishabh-Baranwal) [reply others/create-nginx-vm.sh]: Disabling error on exit will allow the loop to get execute when loop variable is not satisfying the condition specially checking the Null/None with -z. The loop variable results in blank/None with non 0 return code which was making the script to exit. Tried to fix this issue https://avinetworks.atlassian.net/browse/AV-67394

File: others/server-check.sh (hunk 1 above)
Comment (Rishabh-Baranwal) [reply others/server-check.sh]: I believe, we should keep "grep cloud-init" as finished at can be a part of any other log line. I will add --lines 10.

File: others/server-check.sh [hunk 3]
```diff
@@ -1,8 +1,26 @@
...
+        break
+    else
+        echo "Cloud-Init still not finished"
+        sleep_count=$((sleep_count+1))
+        sleep 5
+    fi
+done
+
+set -e
+
```
Comment (anantpatil) [inline others/server-check.sh]: Once the cloud init is done, I think we should wait for some time like 10 seconds for nginx to come up completely...
can you put a sleep of 10s here?

Comment (Rishabh-Baranwal) [reply others/server-check.sh]: Addressed
